packages/zosuss/src/Shell.ts

```diff
@@ -87,16 +91,22 @@ export class Shell {
                                 isUserCommand = true;
                             }
 
-                            if (isUserCommand && dataToPrint.match(/\$ exit/)) {
+                            if (isUserCommand && dataToPrint.includes("$ exit")) {
                                 // if exit found, print out stuff before exit, then stop printing out.
                                 dataToPrint = dataToPrint.slice(0, dataToPrint.indexOf("$ exit"));
                                 stdoutHandler(dataToPrint);
                                 dataToPrint = "";
                                 isUserCommand = false;
-                            } else if (isUserCommand) {
-                                // print out the user command result
-                                stdoutHandler(dataToPrint);
-                                dataToPrint = "";
+                            }
+                            else if (dataToPrint.includes(Shell.startCmdFlag)) {
+                                commandFound = true;
```

Comment:
The `commandFound` variable seems like it may be redundant with `isUserCommand`. Curious if it could be removed?